eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Language.hs

```diff
@@ -1,53 +1,8 @@
 {-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE FlexibleContexts #-}
 
--- | This module provides data structures and operations for talking about
---     Non-native Script languages. It is expected that new languages (or new
---     versions of old languages) will be added here.
-module Cardano.Ledger.Alonzo.Language where
+-- | This module exports the module Cardano.Ledger.Language. That module exports data
+--   structures and operations for talking about Non-native Script languages. It is expected that new languages (or new
+--   versions of old languages) will be added there, and just imported here.
+module Cardano.Ledger.Alonzo.Language (module Cardano.Ledger.Language) where
```

Comment:
is there a reason to keep `Cardano.Ledger.Alonzo.Language`?

Reply:
Only because there are about 40 imports. I didn't want to touch 40 files, and I don't know if consensus or api import these things.

Reply:
We will fix this in a follow on PR, that only changes those 40 imports.